extending-valtimo/form-flow/form-flow.md

```diff
@@ -0,0 +1,3 @@
+# Form flows
```

Comment:
Wondering why this has to be a separate page if all it does is contain a single link to another page.  Maybe we should include the Whitelisting Spring beans page as a section here. We can always move it to its own page if it seems needed in the future.

Reply:
For now I'd just like to stick to the structure. Maybe we can come up with a guideline for it in the future.

Reply:
As a small addition: We know custom step types can be created, this would go here as well. It's just that there is no proper support for this in FE yet.